snuba/datasets/configuration/generic_metrics/entities/sets.yaml

```diff
@@ -0,0 +1,102 @@
+version: v1
+kind: entity
+name: generic_metrics_sets
```

Comment:
Might be a dumb question, but why is this config name `sets.yaml` and why is the entity called `generic_metric_sets` as opposed to `generic_metrics`

Reply:
The entity is the aspect of the product data model that you actually query. The dataset is `generic_metrics` which is really a container for entities and storages.

It's described in some detail on this page: https://getsentry.github.io/snuba/architecture/datamodel.html

If that's a typo and you're wondering why it's named `sets.yaml` vs. `generic_metrics_sets.yaml` it's just because generic_metrics is already in the path -- I'm OK with changing that though if we think it makes the code more discoverable